src/ert/gui/simulation/ensemble_smoother_panel.py

```diff
@@ -48,9 +49,13 @@ def __init__(
     ) -> None:
         super().__init__(EnsembleSmoother)
         self.notifier = notifier
+        self.setObjectName("ensemble_smoother_panel")
 
         layout = QFormLayout()
-        self.setObjectName("ensemble_smoother_panel")
+        lab = QLabel(" ".join(EnsembleSmoother.__doc__.split()))  # type: ignore
```

Comment:
There is a property `EnsembleSmoother.name`, should be usable instead of relying on docstrings

Reply:
This is how all the other panels produce their content.
I could change this to name though, having this one panel act differently.